### PR TITLE
feat: Add devops pipeline custom pipeline modal locales

### PR DESCRIPTION
**Pull request by yazhouio**
### What type of PR is this?

/kind bug

### What this PR does / why we need it:
feat: Add devops pipeline custom pipeline modal locales
### Which issue(s) this PR fixes:

Fixes #https://github.com/kubesphere/ks-devops/issues/609 

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note
feat: Add devops pipeline custom pipeline modal locales
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: locales/en/l10-project-devops-pipeline-detail.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  'Target Pipeline Name': 'Target Pipeline Name',
+  'Quiet Period': 'Quiet Period',
+  'Wait For Completion': 'Wait For Completion',
+  'Propagate Errors': 'Propagate Errors',
+  'Error Message': 'Error Message',
+  Time: 'Time',
+  Unit: 'Unit',
+  'Test Results Location': 'Test Results Location',
+  'Allow Empty Results': 'Allow Empty Results',
+  'Keep Long Output': 'Keep Long Output',
```
Comment (Bettygogo2021) [inline locales/en/l10-project-devops-pipeline-detail.js:38]: Retain long output

File: locales/en/l10-project-devops-pipeline-detail.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  'Skip Publishing Checks': 'Skip Publishing Checks',
```
Comment (Bettygogo2021) [inline locales/en/l10-project-devops-pipeline-detail.js:39]: Skip publishing checks

File: locales/en/l10-project-devops-pipeline-detail.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  'Retry Count': 'Retry Count',
+  Message: 'Message',
+  Submitter: 'Submitter',
+  'Config Name': 'Config Name',
```
Comment (Bettygogo2021) [inline locales/en/l10-project-devops-pipeline-detail.js:43]: Configuration Item

**Review by Bettygogo2021 — CHANGES_REQUESTED** [review]

File: locales/en/l10-project-devops-pipeline-detail.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  'AbortPipeline if quality gate status is not green':
```
[the remark this answers is not in the document]
Comment (yazhouio) [reply locales/en/l10-project-devops-pipeline-detail.js:44]: The key is consistent with the value in YAML, so it will not be changed.

**Pull request merged**